### PR TITLE
Update .NET workload for CentOS Stream 10

### PR DESCRIPTION
**Pull request by omajid**
- Add .NET 8
- Drop .NET 6, and 7

### CODE REVIEW

File: configs/sst_dotnet-dotnet.yaml [hunk 1]
```diff
@@ -2,24 +2,14 @@ document: feedback-pipeline-workload
 version: 1
 data:
   name: .NET
-  description: .NET and .NET Core SDK, tools and runtime
+  description: .NET SDK, tools and runtime
   maintainer: sst_dotnet
 
-  packages: []
-  
-  arch_packages:
```
Comment (omajid) [inline configs/sst_dotnet-dotnet.yaml:10]: .NET 7 and later are available for all 4 architectures that Fedora/RHEL care about.

**Review by yselkowitz — CHANGES_REQUESTED** [review]
These config files are currently tagged both `eln` and `c9s`.  Therefore, the configs must be split, with the removals applying only to the eln config.

**Comment by omajid** [comment]
> These config files are currently tagged both eln and c9s. Therefore, the configs must be split, with the removals applying only to the eln config.

Do we care about `c9s` in the context of minimization? Should I just remove `c9s` instead?

**Comment by yselkowitz** [comment]
> Do we care about `c9s` in the context of minimization? Should I just remove `c9s` instead?

We are continuing to track c9s in CR at this time.  As these removals must only apply to ELN, the configs must be split, with the removals applying only to the eln config.  Of course, if/when dotnet8.0 is added to c9s as well, then it should also be added to the c9s configs, but the previous version, having already been shipped, must not be removed.

**Comment by omajid** [comment]
Thanks for clarifying that. I have split up eln and c9s configuration.

File: configs/sst_dotnet-unwanted-eln.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+document: feedback-pipeline-unwanted
+version: 1
+data:
+  name: .NET Unwanted Packages
+  description: Packages that .NET maintainers do not wish to maintain in RHEL 9
```
Comment (yselkowitz) [inline configs/sst_dotnet-unwanted-eln.yaml:5]:
```suggestion
  description: Packages that .NET maintainers do not wish to maintain in RHEL 10
```

**Review by yselkowitz — CHANGES_REQUESTED** [review]
One small nitpick, looks good otherwise.

**Review by yselkowitz — APPROVED** [review]

**Pull request merged**